tools/gh_release.py

```diff
@@ -20,8 +20,19 @@
 def build_tarball(release_dir, release_tgz, arch):
     """Build a release tarball with local assets"""
     # Do not include signatures in GitHub release since we aren't
-    # making those keys public
-    exclude_files = {"RELEASE_NOTES", "SHA256SUMS.sig"}
+    # making those keys public.
+    # Exclude CPU templates in GitHub release as they are already
+    # available on GitHub without any action (like building a binary).
+    exclude_files = {
+        "RELEASE_NOTES",
+        "SHA256SUMS.sig",
+        "c3.json",
```

Comment:
Could we do `*[x.stem for x in Path("tests/data/static_cpu_templates").glob("*.json")]`. That way we don't hardcode them.

Reply:
Looks better! I'll try to use it! Thanks.

Reply:
applied your suggestion and confirmed it works! thanks!